CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+
 # 5.15.3
```

Comment:
I just switched this to `5.16.0` in my recently merged PR, can you resolve conflicts. Also, it seems much of this file was reformatted. Please revert all of the changes below `5.15.2`. Thanks!